comp/netflow/server/integration_test.go

```diff
@@ -45,8 +45,20 @@ var setTimeNow = fx.Invoke(func(c Component) {
 	}
 })
 
+func assertFlowEventsCount(t *testing.T, port uint16, srv *Server, packetData []byte, expectedEvents uint64) bool {
+	return assert.EventuallyWithT(t, func(c *assert.CollectT) {
+		err := testutil.SendUDPPacket(port, packetData)
+		require.NoError(c, err, "error sending udp packet")
```

Comment:
FYI, you cannot call `require` inside an EventuallyWithT without failing the whole `eventually` block.

Reply:
@leeavital Indeed, seems to lead `panic: Assertion failed` :/ 

Just replace with `assert` here: 82abffc9b4